src/styles/themes/dark.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-unused-vars */
+import colors from '../colors';
+import SCREENS from '../../SCREENS';
+import {HexColor, ThemeColors} from './ThemeColors';
+
+const darkTheme: ThemeColors = {
+    // Figma keys
+    appBG: colors.darkAppBackground,
+    splashBG: colors.green400,
+    highlightBG: colors.darkHighlightBackground,
+    border: colors.darkBorders,
+    borderLighter: colors.darkDefaultButton,
+    borderFocus: colors.green400,
+    icon: colors.darkIcons,
+    iconMenu: colors.green400,
+    iconHovered: colors.darkPrimaryText,
+    iconSuccessFill: colors.green400,
+    iconReversed: colors.darkAppBackground,
+    iconColorfulBackground: `${colors.ivory as HexColor}cc`,
+    textSupporting: colors.darkSupportingText,
+    text: colors.darkPrimaryText,
+    textColorfulBackground: colors.ivory,
+    link: colors.blue300,
+    linkHover: colors.blue100,
+    buttonDefaultBG: colors.darkDefaultButton,
+    buttonHoveredBG: colors.darkDefaultButtonHover,
+    buttonPressedBG: colors.darkDefaultButtonPressed,
+    danger: colors.red,
+    dangerHover: colors.redHover,
+    dangerPressed: colors.redHover,
+    warning: colors.yellow400,
+    success: colors.green400,
+    successHover: colors.greenHover,
+    successPressed: colors.greenPressed,
+    transparent: colors.transparent,
+    signInPage: colors.green800,
+
+    // Additional keys
+    overlay: colors.darkHighlightBackground,
+    inverse: colors.darkPrimaryText,
+    shadow: colors.black,
+    componentBG: colors.darkAppBackground,
+    hoverComponentBG: colors.darkHighlightBackground,
+    activeComponentBG: colors.darkBorders,
+    signInSidebar: colors.green800,
+    sidebar: colors.darkHighlightBackground,
+    sidebarHover: colors.darkAppBackground,
+    heading: colors.darkPrimaryText,
+    textLight: colors.darkPrimaryText,
+    textDark: colors.darkAppBackground,
+    textReversed: colors.lightPrimaryText,
+    textBackground: colors.darkHighlightBackground,
+    textMutedReversed: colors.darkIcons,
+    textError: colors.red,
+    offline: colors.darkIcons,
+    modalBackdrop: colors.darkHighlightBackground,
+    modalBackground: colors.darkAppBackground,
+    cardBG: colors.darkHighlightBackground,
+    cardBorder: colors.darkHighlightBackground,
+    spinner: colors.darkSupportingText,
+    unreadIndicator: colors.green400,
+    placeholderText: colors.darkIcons,
+    heroCard: colors.blue400,
+    uploadPreviewActivityIndicator: colors.darkHighlightBackground,
+    dropUIBG: 'rgba(6, 27, 9, 0.92)',
+    receiptDropUIBG: 'rgba(3, 212, 124, 0.84)',
```

Comment:
Why aren't these colors defined in `colors.ts`?

Reply:
Not sure either, the latest cleanup PR which also introduced light theme didn't change this either. @grgia should we move this to a separate color in `colors.ts` ?